frontend/pages/appointment/helper.js

```diff
@@ -0,0 +1,72 @@
+const generateSchedules = (appointments) => {
```

Comment:
NextJS expects the directory `pages/` to only contain page components. You need to move this file elsewhere.